.github/workflows/wheels.yml

```diff
@@ -1,47 +1,89 @@
-name: Python publication
-
 on:
   release:
     # run only on published, not for drafts
     types: [published]
+  pull_request:
+  workflow_dispatch:
+
+env:
+  flags: --release --out dist --manifest-path pineappl_py/Cargo.toml
 
 jobs:
-  deploy-linux:
+  linux:
     runs-on: ubuntu-latest
-    # a dedicated container is needed, so maturin-action is not an option, see pineappl_py/package/README.md for details
-    container:
-      image: docker://ghcr.io/n3pdf/maturin:0.2.1
+    strategy:
+      matrix:
+        target: [x86_64]
```

Comment:
If you just need a single value, you can also inline this in the place where it is used.
Or do you believe we might need more architectures in the future, and thus it makes sense to keep as a list?

Reply:
Exactly, we might want to support more architectures at some point!